### PR TITLE
Working flags, crimea/taipei, HTTPS, en-dash

### PR DESCRIPTION
**Pull request by comradekingu**
http://www.hsl.fi/EN/passengersguide/travelcard/Pages/default.aspx is dead, but the domain works.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -84,9 +84,9 @@ Card / Agency | Location | Notes
 City Union | :cn: Mainland China | :new:
 [Clipper][clipper] | :us: San Francisco, CA, USA
 [Compass][compass] | :canada: Vancouver, BC, Canada | :new: :one:
-[Crimea Trolleybus][crimea] | Crimea | :new: :closed_lock_with_key: `MFC`
+[Crimea Trolleybus][crimea] | Crimea, Eastern Europe | :new: :closed_lock_with_key: `MFC`
```
Comment (phcoder) [inline README.md]: Same comment as the other patch

File: README.md [hunk 2]
```diff
@@ -84,9 +84,9 @@ Card / Agency | Location | Notes
...
 [Cubic Nextfare][nextfare] | :earth_americas: _many locations_ | :new: :closed_lock_with_key: `MFC`
-[EasyCard][easycard] | :taiwan: Taipei, Taiwan | :closed_lock_with_key: `MFC`
+[EasyCard][easycard] | :taiwan: Taipei, East Asia | :closed_lock_with_key: `MFC`
```
Comment (phcoder) [inline README.md]: "Taiwan" needs to be removed altogether. Feel free to adjust the patch or I'll propose it as a separate patch.

Comment (comradekingu) [reply README.md]: No more Taiwan.

File: README.md (hunk 1 above)
Comment (comradekingu) [reply README.md]: Fixed.

**Comment by micolous** [comment]
Squashed, rebased as 75ae8a905451b4fbb7b5689d9f65fb6923cc6d1d, and merged as 66de0d581df580197084cd9bf4ba96baea508750.

I also updated `docs/index.md` with your changes from `README.md`.

Thanks!

**Pull request closed without merging**